public/components/common/globalBreadcrumb/globalBreadcrumb.scss

```diff
@@ -14,18 +14,39 @@ nav.euiBreadcrumbs.wz-global-breadcrumb span.euiBreadcrumb[title=""]:first-child
   padding: 0px;
 }
 
-.wz-global-breadcrumb-btn{
-  max-width: unset!important;
+.wz-global-breadcrumb-btn {
+  max-width: unset !important;
 }
 
 @media screen and (-webkit-min-device-pixel-ratio: 0) {
-  _::-webkit-full-page-media, _:future, :root , .wz-global-breadcrumb {
+
+  _::-webkit-full-page-media,
+  _:future,
+  :root,
+  .wz-global-breadcrumb {
     line-height: 1.5;
   }
 }
 
 @-moz-document url-prefix() {
   .wz-global-breadcrumb #breadcrumbNoTitle {
-    height: auto!important;
+    height: auto !important;
   }
 }
+
+// styles for opensearch >2.x
+
+.osdHeaderBreadcrumbs,
+.osdHeaderBreadcrumbs--dark {
+  filter: none;
+}
+
+.osdHeaderBreadcrumbs .euiBreadcrumb:not(.euiBreadcrumb:last-child),
+.osdHeaderBreadcrumbs--dark .euiBreadcrumb:not(.euiBreadcrumb:last-child) {
+  margin-right: 8px;
+}
+
+.osdHeaderBreadcrumbs .euiBreadcrumbSeparator,
+.osdHeaderBreadcrumbs--dark .euiBreadcrumbSeparator {
+  display: block;
```

Comment:
**question**: are these classes of OpenSearch Dashboards? If so, then we could are modifying them and it could have side effects. Could we get the solution without modifying the original behaviour?

Reply:
No,doesn't exists a way. Because the osdHeaderBreadcrumb style is added in the global nav tag.